addons/sourcemod/scripting/mapchooser_extended_sounds.sp

```diff
@@ -141,6 +140,9 @@ public void OnPluginStart()
 	g_SoundFiles = CreateTrie();
 	LoadSounds();
 	HookConVarChange(g_Cvar_SoundSet, SoundSetChanged);
+
+	// Make sure it get updated on every plugin load
+	OnConfigsExecuted();
```

Comment:
it does not executed when you reload the plugin?

if so check if its g_bLate

Reply:
Nop, it's only called once, when the map has loaded, servercfgfile (server.cfg) has been executed, and all plugin configs are done executing.